include/wally_elements.h

```diff
@@ -212,6 +300,46 @@ WALLY_CORE_API int wally_asset_surjectionproof(
     size_t len,
     size_t *written);
 
+/**
+ * Generate an explicit transaction output asset surjection proof.
+ *
+ * :param output_asset: asset id for the output.
+ * :param output_asset_len: Length of ``asset``. Must be ``ASSET_TAG_LEN``.
+ * :param output_abf: Asset blinding factor for the output. Generated randomly for each output.
+ * :param output_abf_len: Length of ``output_abf``. Must be ``BLINDING_FACTOR_LEN``.
+ * :param output_generator: Asset generator from `wally_asset_generator_from_bytes`.
```

Comment:
As above, can't we compute the `output_generator` from `output_asset` and `output_abf`?

Reply:
as above